### DISCUSSION TITLE
point_0738

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#411] [Roy]
"20200616T053651_20200616T054718_T42QZL"
![image](https://user-images.githubusercontent.com/54947195/147896793-25cef1f1-471f-4f5c-8748-fb0bb9ae63a7.png)

"20200323T053639_20200323T054547_T42QZL"
![image](https://user-images.githubusercontent.com/54947195/147896884-b0dae557-3020-407b-b084-c4105c8c2d59.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200616T053651_20200616T054718_T42QZL"
![Captura](https://user-images.githubusercontent.com/86502862/153905314-09cff88a-e93f-40c6-b930-62c4f5ad0774.JPG)
